descriptors/ModuleDescriptor-template.json

```diff
@@ -4,7 +4,7 @@
   "provides": [
     {
       "id": "item-storage",
-      "version": "8.1",
+      "version": "8.2",
```

Comment:
We also need to update API version in item-storage.raml.

Reply:
OK. That's quite error prone to have to specify this in two places when that's not checked by some tool. Updated now - for the three RAML files.

Reply:
Thanks. Yeah, indeed it.